bedrock/mozorg/templates/mozorg/about/index.html

```diff
@@ -8,6 +8,8 @@
 
 {% extends "base-protocol-mozilla.html" %}
 
+{% set show_pocket_banner = switch('pocket-banner') and (LANG.startswith('en') or LANG == 'de') %}
```

Comment:
Another option here would be to include a list of locales as an optional parameter in the `switch()` function:
```suggestion
{% set show_pocket_banner = switch('pocket-banner', ['en-US', 'en-CA', 'en-GB', 'de') %}
```

More explicit, but maybe reads a little better?

Reply:
much better, done